packages/tools/examples/videoTools/index.ts

```diff
@@ -226,7 +475,11 @@ async function run() {
   toolGroup.setToolActive(StackScrollTool.toolName, {
     bindings: [
       {
-        mouseButton: MouseBindings.Primary, // Left Click
+        mouseButton: MouseBindings.Secondary,
```

Comment:
Still it is unusually that scroll does not scroll in the video

Reply:
On what button?  The primary is already used.